src/appleseed/renderer/modeling/project/project.xsd

```diff
@@ -294,22 +294,6 @@
                         </xsd:complexContent>
                     </xsd:complexType>
                 </xsd:element>
-                <xsd:element name="rules" minOccurs="0">

```

Comment:
I'm not sure we want to remove them from the schema just yet because that prevents loading scenes with render layer rules.